scripts/generate-css-constants.ts

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env -S deno run --allow-read --allow-write
+
+// Generate TypeScript constants from CSS file
+const cssContent = Deno.readTextFileSync('src/github-patcher.css');
+
+// Create the TypeScript content with the CSS as a string constant
+const tsContent = `// This file is auto-generated by generate-css-constants.ts
+// Do not edit manually
+
+export const HIDE_CSS_CONTENT = \`${cssContent}\`;
+
+export const SHOW_CSS_CONTENT = \`/* Override static CSS to show elements when extension is off - using html prefix for higher specificity */
+${cssContent.split('\n').map(line => {
+  const trimmed = line.trim();
+  if (trimmed === '' || trimmed.startsWith('/*') || trimmed.endsWith('*/')) {
+    return line;
+  }
+  // Add html prefix to selector lines (lines ending with comma or {)
+  if ((trimmed.endsWith(',') || trimmed.endsWith('{')) && !trimmed.startsWith('html ')) {
+    return 'html ' + line;
```

Comment:
Hmm... adding `html` from base CSS seems not simple code. So how about adding `html` prefix added CSS file? And other CSS remove the prefix in generator scripts.

Reply:
I said the generator should generate github-patcher.css from github-patcher-show.css. Did you do that?

Reply:
And so, `github-patcher.css` now should be renamed to `github-patcher-hide.css`